configuration/concepts/patient_document.csv

```diff
@@ -1,4 +1,4 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
-81f330b5-3f10-11e4-adec-0800271c1b75,,,Document,Document,,Misc,Text,,,,,,,,,,,
-c4694ad6-3f10-11e4-adec-0800271c1b75,,,Patient file,Patient file,,Misc,N/A,,,,,,,,,,,
-c46896fd-3f10-11e4-adec-0800271c1b75,,,Patient Document,Patient Document,,ConvSet,N/A,,,Patient file,,,,,,,,
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
+81f330b5-3f10-11e4-adec-0800271c1b75,,,Document,Document,Document,Document,,Misc,Text,,,,,,,,,,,
+c4694ad6-3f10-11e4-adec-0800271c1b75,,,Patient file,Dossier patient,Patient file,Dossier patient,,Misc,N/A,,,,,,,,,,,
+c46896fd-3f10-11e4-adec-0800271c1b75,,,Patient Document,Document patient,Patient Document,Document patient,,ConvSet,N/A,,,Dossier patient,,,,,,,,
```

Comment:
Did you try the document upload? (just to make sure)

Reply:
Fixed...The french name expected is still `Patient Document`